### ISSUE TITLE
about volume location not at centre  at controlBar: {volumePanel: {inline: false}}, at 8.0.3 in chinese

### ISSUE THREAD

**Issue by friendtasy** [videojs/video.js#8092]
### Description

volume location not at centre  at controlBar: {volumePanel: {inline: false}}, at 8.0.3 in chinese
<img width="299" alt="屏幕截图 2023-01-24 152316" src="https://user-images.githubusercontent.com/17196859/214234052-e05b1eb0-dbb8-4f03-ad3d-042a09c718b2.png">
[Uploading test.txt…]()

### Steps to reproduce

1.play video in chinese system and explorer
2.
3.

### What version of Video.js are you using?

8.0.3

### What browser(s) including version(s) does this occur with?

chrome 109 zh-cn

### What OS(es) and version(s) does this occur with?

windows11

**Comment by friendtasy** [comment]
> It probably needs this to be changed to line height as was done for the horizontal bar in #7990
> 
> https://github.com/videojs/video.js/blob/509b3d075750be689ded67ca887d70af578c7804/src/css/components/_volume.scss#L133
> 
> Would you be able to test that?

i don't know how to do this way.you can provide js command change  that,let me try it...use up test.html file

**Comment by mister-ben** [comment]
Manually changing styles on the element in developer tools is probably the easiest way to work it out. I had a try with Chrome Canary in Chinese, but I haven’t found a value that works for both Chinese and non-Chinese browsers.

**Comment by gkatsev** [comment]
> Manually changing styles on the element in developer tools is probably the easiest way to work it out. I had a try with Chrome Canary in Chinese, but I haven’t found a value that works for both Chinese and non-Chinese browsers.

@mister-ben what about targeting css based on locale like we do for the subscaps button?

**Comment by mister-ben** [comment]
No, we can't target it with `:lang()` because it's not the language of the page or part of the DOM that matters; it's the language the browser is runnning in. I can replicate only by [running Chrome in Chinese](https://support.apple.com/en-gb/guide/mac-help/mh26684/mac#mchlf6cd6fbc). It's weird.

**Comment by mister-ben** [comment]
I see the reason it's different now. If Chrome is running in Chinese, it sets the minimum font size to 12px, (in preferences > appearance > customise font). In other languages it's unset. Our CSS sets the slider handle to .9em, which we expect to be 9px, but on the Chinese browser it's 12px becuase of that min value.

**Comment by friendtasy** [comment]
> I see the reason it's different now. If Chrome is running in Chinese, it sets the minimum font size to 12px, (in preferences > appearance > customise font). In other languages it's unset. Our CSS sets the slider handle to .9em, which we expect to be 9px, but on the Chinese browser it's 12px becuase of that min value.

i found that at firefox 109 horizontal button has a litle upward,just like follow image
![Uploading 屏幕截图 2023-02-04 150339.png…]()
